build_debian.sh

```diff
@@ -319,6 +319,8 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     python3-distutils       \
     python3-pip             \
     cron                    \
+    libprotobuf-dev         \
+    libgrpc++               \
```

Comment:
Do we truly need dev packages in sonic image?

Reply:
I feel you install them because `pip` need them to build wheel from source code. Then is it possible to prebuilt and minimize the sonic image disk usage?

Reply:
I have updated the commit to change the libprotobuf-dev to the buster lightly version libprotobuf17.  This library is required by the Nokia NDK (device manager) to run on the sonic plaform